package.json

```diff
@@ -20,24 +20,30 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
+    "@google-cloud/storage": "^7.1.0",
     "@nestjs/common": "^10.0.0",
     "@nestjs/config": "^3.1.1",
     "@nestjs/core": "^10.0.0",
     "@nestjs/platform-express": "^10.0.0",
+    "@nestjs/swagger": "^7.1.13",
     "@nestjs/typeorm": "^10.0.0",
+    "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
     "firebase-admin": "^11.10.1",
     "pg": "^8.11.3",
+    "react-toastify": "^9.1.3",
```

Comment:
Hmm, so this is a FE project or BE?

Reply:
Yeah, I don't know how it got here, but it's been removed now. 😐